domains/blooket.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "email": "",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.L3V3MOMY79wT_NK9PY8vOHLtKZRW2sxVsd-opDX1Wb3R3ZH72mB6C1FsVBSrbnGOJkC-OoEAq3xpLuSOm7KPzdIF9nExBRNltWyCVIGsZb_IGqrKLNxHNacD4Ls-XzVMZncux-ZXgd4-99kPg4NdXugZV34ugk38TQwuqwnRoLKMRjq4WjTP8sS3tidrSVjAZvcGbWbYQVeSo_IDjdGkFhw8Wsur4FFSGB7W2yBU6u1h4C6eYZYjHw5xPafXqzX-IDtMukfgnI1fXIjBgszRUOUuDQEDr4kwL-EecFeA9c1tQG3cQqGFbrt71fINIWq5CH2OsCBvi3dZboc2KsQ2vw.0MVCkLczAR3HwNs5rHRyag.F5TEtKQ3vAwxpynNWdsv0H4g2-Vs1zijaYtnFjM7muDSTQGa-bmwFkIejrlaOVvGuWhq1D4FQsQU_WnMwToCpEwfMPAbif8n80sQfqLj7LOMsD1uzFwE_iBfkdrj_uuI.EJMYfqdvwlqVh5q0GcGnaA"
+        },
+    
+        "record": {
+            "CNAME": "https://xgeneric.github.io/blooket/"
```

```suggestion
            "CNAME": ":xgeneric.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).

---

You had a typo, fixed that now

---

> CNAME records cannot contain protocols (`https://`) or slashes (`/`).

thanks, fixed.